tests/systemtests/test_runner.go

```diff
@@ -127,3 +130,39 @@ func printResultFlag(ok bool) {
 		fmt.Println(failureFlag)
 	}
 }
+
+type GRPCTestCase struct {
+	name   string
+	url    string
+	expOut string
+}
+
+// RunGRPCQueries runs given grpc testcases by making requests and
+// checking response with expected output
+func RunGRPCQueries(t *testing.T, testCases []GRPCTestCase) {
+	t.Helper()
+
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+			resp, err := testutil.GetRequest(tc.url)
+			require.NoError(t, err)
+			require.Contains(t, string(resp), tc.expOut)
```

Comment:
We might not always keep complete expected output as sometimes output might be huge. So, I thought of using Contains so that we can check in response whatever required for us.